build/commands/scripts/commands.js

```diff
@@ -166,6 +167,19 @@ program
   .arguments('[build_config]')
   .action(build)
 
+program
+  .command('build_chromium_release')
+  .option('--force', 'Ignore a warning about non-CI build')
+  .option('-C <build_dir>', 'build config (out/Debug, out/Release')
```

Comment:
I don't know if we should have this option here? People might accidentally overwrite the output dir for Brave? Can we also add some comments explaining what this does/how this works?

Reply:
Discussed in DM. It allows to use another dir with custom gn args
Tiny chance to ruin `out/Release`